# HG changeset patch
# User asniker
# Date 1286782209 -21600
# Node ID 2532ba10223723b560466a4b8f6e3b2f8c3a700c
# Parent  9f78b7fbb0cf426c71c9998dfdabde5b8e0053e8
fix chat.cpp and chat.h and add updates sql

diff --git a/sql/updates/10189_world_script_texts.sql b/sql/updates/10189_world_script_texts.sql
new file mode 100644
--- /dev/null
+++ b/sql/updates/10189_world_script_texts.sql
@@ -0,0 +1,15 @@
+DELETE FROM `script_texts` WHERE `entry` BETWEEN -1631103 AND -1631091;
+INSERT INTO `script_texts` (`npc_entry`,`entry`,`content_default`,`content_loc1`,`content_loc2`,`content_loc3`,`content_loc4`,`content_loc5`,`content_loc6`,`content_loc7`,`content_loc8`,`sound`,`type`,`language`,`emote`,`comment`) VALUES
+(36678,-1631091,'Great news, everyone! The slime is flowing again!',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,17126,1,0,1,'SAY_ROTFACE_OOZE_FLOOD1'),
+(36678,-1631092,'Good news, everyone! I''ve fixed the poison slime pipes!',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,17123,1,0,1,'SAY_ROTFACE_OOZE_FLOOD2'),
+(36678,-1631093,'Terrible news, everyone, Rotface is dead! But great news everyone, he left behind plenty of ooze for me to use! Whaa...? I''m a poet, and I didn''t know it? Astounding!',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,17146,1,0,0,'SAY_ROTFACE_DEATH'),
+(36627,-1631094,'What? Precious? Noooooooooo!!!',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,16993,1,0,0,'SAY_PRECIOUS_DIES'),
+(36627,-1631095,'WEEEEEE!',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,16986,1,0,0,'SAY_AGGRO'),
+(36627,-1631096,'%s begins to cast Slime Spray!',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,0,3,0,0,'EMOTE_SLIME_SPRAY'),
+(36627,-1631097,'Icky sticky.',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,16991,1,0,0,'SAY_SLIME_SPRAY'),
+(36627,-1631098,'|TInterface\Icons\spell_shadow_unstableaffliction_2.blp:16|t%s begins to cast |cFFFF0000Unstable Ooze Explosion!|r',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,0,3,0,0,'EMOTE_UNSTABLE_EXPLOSION'),
+(36627,-1631099,'I think I made an angry poo-poo. It gonna blow!',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,16992,1,0,0,'SAY_UNSTABLE_EXPLOSION'),
+(36627,-1631100,'Daddy make toys out of you!',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,16988,1,0,0,'SAY_KILL_1'),
+(36627,-1631101,'I brokes-ded it...',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,16987,1,0,0,'SAY_KILL_2'),
+(36627,-1631102,'Sleepy Time!',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,16990,1,0,0,'SAY_BERSERK'),
+(36627,-1631103,'Bad news daddy...',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,16989,1,0,0,'SAY_DEATH');
diff --git a/sql/updates/10189_world_scriptname.sql b/sql/updates/10189_world_scriptname.sql
new file mode 100644
--- /dev/null
+++ b/sql/updates/10189_world_scriptname.sql
@@ -0,0 +1,4 @@
+UPDATE `creature_template` SET `ScriptName`='boss_rotface' WHERE `entry`=36627;
+UPDATE `creature_template` SET `ScriptName`='npc_little_ooze' WHERE `entry`=36897;
+UPDATE `creature_template` SET `ScriptName`='npc_big_ooze' WHERE `entry`=36899;
+UPDATE `creature_template` SET `ScriptName`='npc_precious_icc' WHERE `entry`=37217;
diff --git a/sql/updates/10189_world_spell_linked_spell.sql b/sql/updates/10189_world_spell_linked_spell.sql
new file mode 100644
--- /dev/null
+++ b/sql/updates/10189_world_spell_linked_spell.sql
@@ -0,0 +1,6 @@
+DELETE FROM `spell_linked_spell` WHERE `spell_effect`=69706;
+INSERT INTO `spell_linked_spell` (`spell_trigger`,`spell_effect`,`type`,`comment`) VALUES
+(-69674,69706,0, 'Rotface: Mutated Infection Summon'),
+(-71224,69706,0, 'Rotface: Mutated Infection Summon'),
+(-73022,69706,0, 'Rotface: Mutated Infection Summon'),
+(-73023,69706,0, 'Rotface: Mutated Infection Summon');
diff --git a/sql/updates/10189_world_spell_script_names.sql b/sql/updates/10189_world_spell_script_names.sql
new file mode 100644
--- /dev/null
+++ b/sql/updates/10189_world_spell_script_names.sql
@@ -0,0 +1,18 @@
+DELETE FROM `spell_script_names` WHERE `spell_id` IN (69782,69796,69798,69801) AND `ScriptName`='spell_rotface_ooze_flood';
+DELETE FROM `spell_script_names` WHERE `spell_id`=69538 AND `ScriptName`='spell_rotface_little_ooze_combine';
+DELETE FROM `spell_script_names` WHERE `spell_id`=69553 AND `ScriptName`='spell_rotface_large_ooze_combine';
+DELETE FROM `spell_script_names` WHERE `spell_id`=69610 AND `ScriptName`='spell_rotface_large_ooze_buff_combine';
+DELETE FROM `spell_script_names` WHERE `spell_id`=69839 AND `ScriptName`='spell_rotface_unstable_ooze_explosion_init';
+DELETE FROM `spell_script_names` WHERE `spell_id`=69832 AND `ScriptName`='spell_rotface_unstable_ooze_explosion';
+DELETE FROM `spell_script_names` WHERE `spell_id`=71441 AND `ScriptName`='spell_rotface_unstable_ooze_explosion_suicide';
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
+(69782,'spell_rotface_ooze_flood'),
+(69796,'spell_rotface_ooze_flood'),
+(69798,'spell_rotface_ooze_flood'),
+(69801,'spell_rotface_ooze_flood'),
+(69538,'spell_rotface_little_ooze_combine'),
+(69553,'spell_rotface_large_ooze_combine'),
+(69610,'spell_rotface_large_ooze_buff_combine'),
+(69839,'spell_rotface_unstable_ooze_explosion_init'),
+(69832,'spell_rotface_unstable_ooze_explosion'),
+(71441,'spell_rotface_unstable_ooze_explosion_suicide');
diff --git a/sql/updates/10192_world_creature.sql b/sql/updates/10192_world_creature.sql
new file mode 100644
--- /dev/null
+++ b/sql/updates/10192_world_creature.sql
@@ -0,0 +1,4 @@
+ALTER TABLE `creature`
+ ADD COLUMN `npcflag` int(10) unsigned NOT NULL DEFAULT '0' AFTER `MovementType`,
+ ADD COLUMN `unit_flags` int(10) unsigned NOT NULL DEFAULT '0' AFTER `npcflag`,
+ ADD COLUMN `dynamicflags` int(10) unsigned NOT NULL DEFAULT '0' AFTER `unit_flags`;
diff --git a/sql/updates/10195_world_script_texts.sql b/sql/updates/10195_world_script_texts.sql
new file mode 100644
--- /dev/null
+++ b/sql/updates/10195_world_script_texts.sql
@@ -0,0 +1,30 @@
+DELETE FROM `script_texts` WHERE `entry` IN (-1575019,-1575020);
+INSERT INTO `script_texts` (`npc_entry`,`entry`,`content_default`,`content_loc1`,`content_loc2`,`content_loc3`,`content_loc4`,`content_loc5`,`content_loc6`,`content_loc7`,`content_loc8`,`sound`,`type`,`language`,`emote`,`comment`) VALUES
+(26668,-1575019,'You were a fool to challenge the power of the Lich King!',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,0,0,0,0,'Svala Sorrowgrave SAY_SLAY_1'),
+(26668,-1575020,'Your will is done, my king.',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,0,0,0,0,'Svala Sorrowgrave SAY_SLAY_2');
+
+DELETE FROM `script_texts` WHERE `entry` IN (-1604002,-1604009);
+INSERT INTO `script_texts` (`npc_entry`,`entry`,`content_default`,`content_loc1`,`content_loc2`,`content_loc3`,`content_loc4`,`content_loc5`,`content_loc6`,`content_loc7`,`content_loc8`,`sound`,`type`,`language`,`emote`,`comment`) VALUES
+(29306,-1604002,'Who needs gods, when WE ARE GODS!',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,0,0,0,0,'Gal''darah SAY_SLAY_2'),
+(29306,-1604009,'You wanna see power? I''m gonna show you power!',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,0,0,0,0,'Gal''darah SAY_TRANSFORM_2');
+
+DELETE FROM `script_texts` WHERE `entry` IN (-1619003,-1619004,-1619005,-1619007,-1619010,-1619011,-1619012,-1619013);
+INSERT INTO `script_texts` (`npc_entry`,`entry`,`content_default`,`content_loc1`,`content_loc2`,`content_loc3`,`content_loc4`,`content_loc5`,`content_loc6`,`content_loc7`,`content_loc8`,`sound`,`type`,`language`,`emote`,`comment`) VALUES
+(29310,-1619003,'Yogg-Saron, grant me your power!',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,0,0,0,0,'Jedoga Shadowseeker SAY_SACRIFICE_2_1'),
+(29310,-1619004,'Master, a gift for you!',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,0,0,0,0,'Jedoga Shadowseeker SAY_SACRIFICE_2_2'),
+(29310,-1619005,'Glory to Yogg-Saron!',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,0,0,0,0,'Jedoga Shadowseeker SAY_SLAY_1'),
+(29310,-1619007,'Get up! You haven''t suffered enough.',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,0,0,0,0,'Jedoga Shadowseeker SAY_SLAY_3'),
+(29310,-1619010,'Immortality can be yours. But only if you pledge yourself fully to Yogg-Saron!',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,0,0,0,0,'Jedoga Shadowseeker SAY_PREACHING_2'),
+(29310,-1619011,'Here on the very borders of his domain. You will experience powers you would never have imagined!',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,0,0,0,0,'Jedoga Shadowseeker SAY_PREACHING_3'),
+(29310,-1619012,'You have traveled long and risked much to be here. Your devotion shall be rewarded.',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,0,0,0,0,'Jedoga Shadowseeker SAY_PREACHING_4'),
+(29310,-1619013,'The faithful shall be exalted! But there is more work to be done. We will press on until all of Azeroth lies beneath his shadow!',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,0,0,0,0,'Jedoga Shadowseeker SAY_PREACHING_5');
+
+DELETE FROM `script_texts` WHERE `entry` IN (-1619020);
+INSERT INTO `script_texts` (`npc_entry`,`entry`,`content_default`,`content_loc1`,`content_loc2`,`content_loc3`,`content_loc4`,`content_loc5`,`content_loc6`,`content_loc7`,`content_loc8`,`sound`,`type`,`language`,`emote`,`comment`) VALUES
+(29309,-1619020,'Shhhad ak kereeesshh chak-k-k!',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,0,0,0,0,'Elder Nadox SAY_EGG_SAC_2');
+
+DELETE FROM `script_texts` WHERE `entry` IN (-1619031,-1619032,-1619033);
+INSERT INTO `script_texts` (`npc_entry`,`entry`,`content_default`,`content_loc1`,`content_loc2`,`content_loc3`,`content_loc4`,`content_loc5`,`content_loc6`,`content_loc7`,`content_loc8`,`sound`,`type`,`language`,`emote`,`comment`) VALUES
+(29311,-1619031,'Ywaq puul skshgn: on''ma yeh''glu zuq.',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,0,0,0,0,'Herald Volazj SAY_SLAY_1'),
+(29311,-1619032,'Ywaq ma phgwa''cul hnakf.',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,0,0,0,0,'Herald Volazj SAY_SLAY_2'),
+(29311,-1619033,'Ywaq maq oou; ywaq maq ssaggh. Ywaq ma shg''fhn.',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,0,0,0,0,'Herald Volazj SAY_SLAY_3');
diff --git a/src/server/game/Chat/Chat.cpp b/src/server/game/Chat/Chat.cpp
--- a/src/server/game/Chat/Chat.cpp
+++ b/src/server/game/Chat/Chat.cpp
@@ -711,27 +711,9 @@
         { NULL,             0,                  false, NULL,                                            "", NULL }
     };
 	
-	static ChatCommand displaysetCommandTable[] =
-	{
-	    {"list",            SEC_PLAYER,         true, &ChatHandler::DisplayListCommand,              "", NULL},
-        {"listitems",       SEC_PLAYER,         true, &ChatHandler::DisplayListItemsCommand,         "", NULL},
-        {"create",          SEC_PLAYER,         true, &ChatHandler::DisplayCreateCommand,            "", NULL},
-        {"delete",          SEC_PLAYER,         true, &ChatHandler::DisplayDeleteCommand,            "", NULL},
-        {"activate",        SEC_PLAYER,         true, &ChatHandler::DisplayActivateCommand,          "", NULL},
-        {"disable",         SEC_PLAYER,         true, &ChatHandler::DisplayDisableCommand,           "", NULL},
-        {"modify",          SEC_PLAYER,         true, &ChatHandler::DisplayModifyCommand,            "", modifyCommandTable}
-	};
-	
-	static ChatCommand modifyCommandTable[] =
-	{
-	    {"enchantment"      SEC_PLAYER,         true, &ChatHandler::DisplayModifyEnchantmentCommand,  "", NULL},
-		{"name"             SEC_PLAYER,         true, &ChatHandler::DisplayModifyNameCommand,         "", NULL}
-	}; 
-
     static ChatCommand commandTable[] =
     {
         { "account",        SEC_PLAYER,         true,  NULL,                                           "", accountCommandTable  },
-		{ "displayset",     SEC_PLAYER,         true,  NULL,                                           "", displaysetCommandTable  },
         { "achievement",    SEC_ADMINISTRATOR,  false, NULL,                                          "", achievementCommandTable},
         { "gm",             SEC_MODERATOR,      true,  NULL,                                           "", gmCommandTable       },
         { "npc",            SEC_MODERATOR,      false, NULL,                                           "", npcCommandTable      },
diff --git a/src/server/game/Chat/Chat.h b/src/server/game/Chat/Chat.h
--- a/src/server/game/Chat/Chat.h
+++ b/src/server/game/Chat/Chat.h
@@ -453,18 +453,7 @@
         bool HandleReloadWpScriptsCommand(const char* args);
         bool HandleReloadConditions(const char* args);
         bool HandleReloadCreatureText(const char* args);
-        //The Displayset.....
-		bool DisplayListCommand(const char* args);
-		bool DisplayListItemsCommand(const char* args);
-		bool DisplayCreateCommand(const char* args);
-		bool DisplayDeleteCommand(const char* args);
-		bool DisplayActivateCommand(const char* args);
-		bool DisplayDisableCommand(const char* args);
-		bool DisplayModifyCommand(const char* args);
-		//The Displayset modify...
-		bool DisplayModifyEnchantmentCommand (const char* args);
-		bool DisplayModifyNameCommand(const char* args);
-		
+       
         bool HandleResetAchievementsCommand(const char * args);
         bool HandleResetAllCommand(const char * args);
         bool HandleResetHonorCommand(const char * args);
